data-transfer/doc/rest.md

```diff
@@ -8,34 +8,34 @@ Consumer initiates flow by sending TransferRequestMessage to provider endpoint h
 ```
 {
 "@context":  "https://w3id.org/dspace/2024/1/context.json",
-"@type": "dspace:TransferRequestMessage",
-"dspace:consumerPid": "urn:uuid:CONSUMER_PID_TRANSFER",
-"dspace:agreementId": "urn:uuid:AGREEMENT_ID",
+"@type": "TransferRequestMessage",
```

Comment:
Should we add section for PUSH transfer?
Check if links to diagrams are correct (might be PR review problem)

Reply:
I agree, it will be added